app/javascript/src/components/Editor.tsx

```diff
@@ -363,6 +375,148 @@ class Editor extends React.Component<EditorProps, EditorState> {
     }
   }
 
+  private onCopy = () => {
+    const { model } = this
+
+    const selectedNodes = model.getSelectedItems().filter(item => {
+      return item instanceof DefaultNodeModel
+    }) as DefaultNodeModel[]
```

Comment:
Style thing, I think this could be cleaned up to use parens, which return their value:

`.filter(item => (
  item instanceOf DefaultNodeModel
)`